### PR TITLE
insight groups plugin - Improve README, make it work against Panther/Tiger

### PR DESCRIPTION
**Pull request by BugsBunny338**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: insight_groups_plugin/README.md [hunk 1]
```diff
@@ -6,18 +6,21 @@ The plugin is grouping multiple widgets into one group. The current selected wid
...
-3. Navigate to `insight_groups_plugin`
-4. Make sure you have your `.env` and `.env.secrets` files with correct values. See [development guide](#Plugin development guide) section.
-5. Make sure that dependencies in `package.json` file are aligned with the version of SDK dashboard component you use in your project.
-6. Build a production version of the plugin with command `npm run build-plugin` or for yarn, `yarn build-plugin`. If you have this plugin already built, delete the `dist` folder first.
-7. Upload built plugin to your hosting. See [limitations for the hosting](https://sdk.gooddata.com/gooddata-ui/docs/dashboard_plugins.html#current-limitations).
-8. Create plugin MD object with `yarn add-plugin` command. For more information run the `yarn add-plugin --help` command. Remember or copy the plugin object id noted in the console output.
-9. Link the plugin to dashboard with the id set up in `.env` file with `yarn link-plugin -- <plugin-obj-id>` command.
+1. Clone [dashboard-plugin-examples repository](https://github.com/gooddata/gooddata-plugin-examples).
+1. Navigate to `insight_groups_plugin`.
```
Comment (kandl) [inline insight_groups_plugin/README.md:10]: All the steps here are step `1.`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: insight_groups_plugin/README.md (hunk 1 above)
Comment (BugsBunny338) [reply insight_groups_plugin/README.md:10]: That is correct. Github markdown will do the numbering for us. This way, when you need to add a step anywhere to the list you don't need to rewrite all the numbers 😉

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-plugin-examples-gate-PR31

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-sonarqube-scan-zuul-gate/32/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-merge-compliance-check/33/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-plugin-examples-fossa-scanning-zuul/9/ : FAILURE in 45s (non-voting)

**Pull request merged**